packages/ssr/src/utils/cookies.ts

```diff
@@ -0,0 +1,77 @@
+import { Session } from '@supabase/supabase-js';
+import { parse, serialize } from 'cookie';
+import { base64url } from 'jose';
+
+export { parse as parseCookies, serialize as serializeCookie };
+
+/**
+ * Based on the environment and the request we know if a secure cookie can be set.
+ */
+export function isSecureEnvironment(headerHost?: string | string[]) {
```

Comment:
@silentworks Is this function still needed? Doesn't look like it is called anywhere

Reply:
Removed 👍